src/core/src/main/java/org/apache/jmeter/report/processor/graph/impl/LatencyOverTimeGraphConsumer.java

```diff
@@ -59,10 +71,17 @@ protected TimeStampKeysSelector createTimeStampKeysSelector() {
     @Override
     protected Map<String, GroupInfo> createGroupInfos() {
         HashMap<String, GroupInfo> groupInfos = new HashMap<>();
-        groupInfos.put(AbstractGraphConsumer.DEFAULT_GROUP, new GroupInfo(
-                new MeanAggregatorFactory(), new NameSeriesSelector(),
-                // We ignore Transaction Controller results
-                new LatencyValueSelector(false), false, false));
+        groupInfos.put("aggregate_report_max", //$NON-NLS-1$
+            createMaxGroupInfo());
+
+        LatencyValueSelector valueSelector = new LatencyValueSelector(false);
+        StaticSeriesSelector seriesSelector = new StaticSeriesSelector();
+        String seriesName = String.format(PERCENTILE_FORMAT, Integer.valueOf(95));
+        seriesSelector.setSeriesName(seriesName);
+
+        groupInfos.put(PERCENTILE_PROPERTY, //$NON-NLS-1$
```

Comment:
I don't think `$NON-NLS-1$` is required here, or below as they are not String literals?

Reply:
Good spot :)